pyccel/stdlib/cwrapper/cwrapper.c

```diff
@@ -0,0 +1,508 @@
+#include "cwrapper.h"
+
+
+
+// strings order needs to be the same as its equivalent numpy macro
+// https://numpy.org/doc/stable/reference/c-api/dtype.html
+const char* dataTypes[17] = {"Bool", "Int8", "UInt8", "Int16", "UIn16", "Int32", "UInt32",
+                             "Int64", "UInt64", "Int128", "UInt128", "Float32", "Float64",
+                             "Float128", "Complex64", "Complex128", "Complex256"};
+
+
+
+/*
+ * Functions : Cast functions
+ * --------------------------
+ * All functions listed down are based on C/python api
+ * with more tolerance to different precision
+ * Convert python type object to the desired C type
+ * Parameters :
+ *     object : the python object
+ * Returns    :
+ *     The desired C type, an error may be raised by c/python converter
+ *     so one should call PyErr_Occurred() to check for errors after the
+ *	   calling a cast function
+ * Reference of the used c python api function
+ * --------------------------------------------
+ * https://docs.python.org/3/c-api/float.html#c.PyFloat_AsDouble
+ * https://docs.python.org/3/c-api/complex.html#c.PyComplex_RealAsDouble
+ * https://docs.python.org/3/c-api/complex.html#c.PyComplex_ImagAsDouble
+ * https://docs.python.org/3/c-api/long.html#c.PyLong_AsLong
+ * https://docs.python.org/3/c-api/long.html#c.PyLong_AsLongLong
+ */
+
+float complex PyComplex_to_Complex64(PyObject *object)
+{
+	float complex	c;
+
+	// https://numpy.org/doc/1.17/reference/c-api.array.html#c.PyArray_IsScalar
+	// https://numpy.org/doc/stable/reference/c-api/array.html#c.PyArray_ScalarAsCtype
+	if (PyArray_IsScalar(object, Complex64))
+		PyArray_ScalarAsCtype(object, &c);
+
```

Comment:
delete this line

Reply:
Why?

Reply:
it's an empty line ^^,` else` should follow directly `if` statement, code standard quality.

Reply:
Ah, yes you are right. I assumed there was no else, but looking at it in context, it is strange

Reply:
Fixed 7d3d8d3